autoscale-api/src/main/resources/messages/messages.properties

```diff
@@ -4,6 +4,7 @@ autoscale.unsupported.hostgroups=Unsupported HostGroup ''{0}'' for {1} Autoscali
 autoscale.config.not.found=''{0}'' Autoscale Configuration with identifier ''{1}'' not found for Cluster ''{2}''.
 autoscale.entitlement.not.enabled=Autoscaling Entitlement is not enabled for CloudPlatform ''{0}'', Cluster ''{1}''.
 autoscale.stopstart.entitlement.not.enabled=Instance Stop/Start scaling mechanism entitlement is not enabled for CloudPlatform ''{0}'', Cluster ''{1}''.
+impala.schedule.based.scaling.entitlement.not.enabled=Impala Schedule based scaling is not entitled for CloudPlatform ''{0}'', Cluster ''{1}''
```

Comment:
required entitlement should also be part of the message

Reply:
Done